src/EA.Weee.Web.Tests.Unit/Areas/AatfReturn/Controller/ObligatedSentOnControllerTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿namespace EA.Weee.Web.Tests.Unit.Areas.AatfReturn.Controller
+{
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Api.Client;
+    using EA.Weee.Core.AatfReturn;
+    using EA.Weee.Core.Helpers;
+    using EA.Weee.Core.Scheme;
+    using EA.Weee.Requests.AatfReturn;
+    using EA.Weee.Requests.AatfReturn.Obligated;
+    using EA.Weee.Web.Areas.AatfReturn.Controllers;
+    using EA.Weee.Web.Areas.AatfReturn.Mappings.ToViewModel;
+    using EA.Weee.Web.Areas.AatfReturn.Requests;
+    using EA.Weee.Web.Areas.AatfReturn.ViewModels;
+    using EA.Weee.Web.Constant;
+    using EA.Weee.Web.Services;
+    using EA.Weee.Web.Services.Caching;
+    using FakeItEasy;
+    using FluentAssertions;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using System.Web.Mvc;
+    using Xunit;
+
+    public class ObligatedSentOnControllerTests
+    {
+        private readonly IWeeeClient weeeClient;
+        private readonly IObligatedSentOnWeeeRequestCreator requestCreator;
+        private readonly BreadcrumbService breadcrumb;
+        private readonly ObligatedSentOnController controller;
+        private readonly IWeeeCache cache;
+        private readonly IMap<ReturnToObligatedViewModelMapTransfer, ObligatedViewModel> mapper;
+        private readonly ICategoryValueTotalCalculator calculator;
+
```

Comment:
Need to add Index GET tests, i.e. that the mapper is called with action parameters and the model returned to the view, that the default view is returned and probably the API is called